doc/community-build/Generate-Cortx-Build-Stack.md

```diff
@@ -63,55 +56,7 @@ To know about various CORTX components, see [CORTX Components guide](https://git
    -rw-r--r--   1 root root 845556896 Apr  9 07:23 third-party-centos-7.8.2003-1.0.0-0.tar.gz
    ```
 
-7. To view each component targets, run:
```

Comment:
Can we keep this help message. This will help users to see the list of targets available.

Reply:
done